docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/matrix/einsum-7.rst

```diff
@@ -111,7 +121,15 @@ Example 5 shows how *Einsum* transposes input tensor:
              [[3.0, 6.0, 9.0]]]
 
 
-In addition to an alphabetic label, ellipsis ``...`` can be used as a label in a subscript to cover broadcasted dimensions. Each input subscript can contain at most one ellipsis. For example, the ellipsis in input subscript ``a...bc`` for five rank tensor covers the second and third dimensions. In case input subscripts contain ellipsis for several operands, the dimensions covered by the ellipsis must be broadcastable to satisfy numpy broadcasting (or multidirectional broadcasting) rules available in :doc:`Broadcast Rules For Elementwise Operations <../../broadcast-rules>`. If at least one input subscript contains an ellipsis, the output subscript must always contain one ellipsis. For example, *Einsum* operation on two inputs of shapes ``[9, 1, 4, 3]`` and ``[3, 11, 7, 1]`` with ``equation="a...b,b...->a..."`` has ellipsis for both operands covering dimensions with sizes ``[1, 4]`` and ``[11, 7, 1]`` that are broadcasted to ``[11, 7, 4]``. The resulted shape of *Einsum* operation will be ``[9, 11, 7, 4]`` since the dimension labeled with ``a`` is left with broadcasted dimensions.
```

Comment:
(Optional) As the note "If at least one input subscript contains an ellipsis, the output subscript must always contain one ellipsis." has been removed, it would be good to add an example to show when it's not true.

Reply:
Added alternative equation to already existing example that show reduced ellipsis + added new example on other axis